modules/wordads/wordads.php

```diff
@@ -64,6 +64,26 @@ function option( $option, $default = false ) {
 		return $this->params->options[ $option ];
 	}
 
+	/**
+	 * Returns the ad tag property array for supported ad types.
+	 * @return array      array with ad tags
+	 *
+	 * @since 7.1.0
+	 */
+	function get_ad_tags() {
```

Comment:
I assume you'll port those to the wpcom side as well, so we can use those functions in both environments?

Reply:
Yeah, I added these functions for code & PHP version compatibility's sake, I'll need to add these on the WP.com site.

Reply:
D24711-code